packages/react/src/ComparisonTable/ComparisonTable.tsx

```diff
@@ -72,6 +73,7 @@ export const _ComparisonTable = forwardRef(
                       )}
                     >
                       {rowChild.props.children}
+                      {isFeatured && <span className="visually-hidden">(featured)</span>}
```

Comment:
qq. is there a way we can make this customizable by end users? This is currently a hard coded English string, which we'll need to eventually change for localization reasons, but there's also an edge case where content may need to change to better describe the table cell header more accurately (feature / highlighted / spotlighted). I think the the flexibility would be nice to offer on the API.. Not something we need to solve now if it balloons the scope though.